jekyll/_cci2/circleci-images.md

```diff
@@ -358,9 +319,18 @@ The following packages are installed via `curl` or other means.
 ## Out of scope
 {: #out-of-scope }
 
-1. If an image isn't listed above, it is not available. As the Convenience Image program is revamped, proposals for new images are not currently being accepted.
-1. Old versions of software will not be rebuilt. Once an upstream image stops building the tag for a specific release, say Node.js v8.1.0, then we stop building it too. This means other tools in that image, such as `npm` in this example, will no longer be updated either.
-1. We don't support building preview, beta, or release candidate images tags. On occasion they'll be available but these tags tend to cause our build system for Convenience Images to fail. If you need a non-stable release of a language, we suggest installing it via [an orb](https://circleci.com/orbs/) or a custom Docker image instead.
+1. If an image isn't listed above, it is not available. As the Convenience Image
+   program is revamped, proposals for new images are not currently being
+   accepted.
+1. Old versions of software will not be rebuilt. Once an upstream image stops
+   building the tag for a specific release, say Node.js v8.1.0, then we stop
+   building it too. This means other tools in that image, such as `npm` in this
+   example, will no longer be updated either.
+1. We don't support building preview, beta, or release candidate images tags. On
+   occasion they'll be available but these tags tend to cause our build system
+   for Convenience Images to fail. If you need a non-stable release of a
+   language, we suggest installing it via [an orb](https://circleci.com/orbs/)
+   or a custom Docker image instead.
```

Comment:
I am wonder ing if the next-generation conveniences images have become commonplace enough to warrant removing this section?